trino-hms-hdfs-ranger/big-data-c3-tests/trino-spark-tests/trino/test_11.sh

```
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+source "./big-data-c3-tests/lib.sh"
+source "./big-data-c3-tests/env_variables.sh"
+
+set -e
+
+echo ""
+echo "## Test 11 ##"
+echo "Attempt various DDL operations against the managed table as a different user"
+echo ""
+
+# It's the same as in the previous test.
+updateHdfsPathPolicy "read,write,execute:$TRINO_USER1,$TRINO_USER2" "/*,/data/projects/gross_test,/$HIVE_WAREHOUSE_DIR/gross_test.db"
+
+updateHiveDbAllPolicy "alter,create,drop,index,lock,select,update:$TRINO_USER1/select:$TRINO_USER2" "gross_test"
+
+# It's the same as in the previous test.
+updateHiveDefaultDbPolicy ""
+
+# It's the same as in the previous test.
+updateHiveUrlPolicy "read,write:$TRINO_USER1" "hdfs://$NAMENODE_NAME/data/projects/gross_test,hdfs://$NAMENODE_NAME/$HIVE_WAREHOUSE_DIR/gross_test.db"
+
+waitForPoliciesUpdate
+
+# Run the commands as user2.
+command="drop table $TRINO_HIVE_SCHEMA.gross_test.test"
+# In the BigData notes, this command is expected to fail with this error. But this is a Spark error.
+# The Trino error for lack of DROP privileges, is different.
+
+# expectedMsg="Permission denied: user [$TRINO_USER2] does not have [DROP] privilege on [gross_test/test]"
+expectedMsg="The following metastore delete operations failed: drop table gross_test.test"
```

Does the trino error specify the reason why the drop failed?  Can you paste the full text of the trino error here in the PR?

---

That's the entire error message. The only extra, is the `Query failed` prefix which is part of every failure. No further explanation.

```
Query 20240704_101923_00021_sjv8x failed: The following metastore delete operations failed: drop table gross_test.test
```

---

If you remember we have already done some research on that. Trino swallows this error `user [$TRINO_USER2] does not have [DROP] privilege on [gross_test/test]` and creates the one that we see. 

https://github.com/G-Research/gr-oss/issues/551